src/NUnitFramework/framework/Internal/Execution/EventListenerTextWriter.cs

```diff
@@ -34,34 +34,42 @@ namespace NUnit.Framework.Internal.Execution
     /// listener is active in the context, or if there is no context,
     /// the output is forwarded to the supplied default writer.
     /// </summary>
-	public class EventListenerTextWriter : TextWriter
-	{
+    public class EventListenerTextWriter : TextWriter
+    {
         private readonly TextWriter _defaultWriter;
-		private readonly string _streamName;
+        private readonly string _streamName;
 
         /// <summary>
         /// Construct an EventListenerTextWriter
         /// </summary>
         /// <param name="streamName">The name of the stream to use for events</param>
         /// <param name="defaultWriter">The default writer to use if no listener is available</param>
-		public EventListenerTextWriter( string streamName, TextWriter defaultWriter )
-		{
-			_streamName = streamName;
+        public EventListenerTextWriter( string streamName, TextWriter defaultWriter )
+        {
+            _streamName = streamName;
             _defaultWriter = defaultWriter;
-		}
+        }
 
         /// <summary>
         /// Get the Encoding for this TextWriter
         /// </summary>
         public override Encoding Encoding { get; } = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
 
+        private string FormatForListener(object value)
+        {
+            return
+                value is null ? string.Empty :
```

Comment:
I have always lined up and indented the ternary op like so...

```c#
return value is null
    ? string.Empty
    : value is IFormattable formattable
        ? formattable.ToString(null, FormatProvider)
        : value.ToString();
```

I'm not sure if we have a standard, but that seems much clearer to me.

I'm as yet unclear about the inline declaration of `formattable`. I suspect I'll like it after I get used to it though.

Reply:
I go back and forth between that formatting and this one depending on how I want to look at it: flat and linear like a switch statement, or as a tree. The way it is now, it lines up all the cases after `?` characters and ends all the lines with an outcome. I usually switch to the tree structure when it can't be thought of as flat, but I see that situation as more complex. Happy to change this though.

Reply:
> Only a suggestion. I could try to look at it as a switch statement and see how that works.

Okay, I might lean towards leaving it as-is then.

Speaking of switching, here is C# 8's [switch expression](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#switch-expressions)—not sure yet if I like it as much as my flat ternary though:

```c#
return value switch
{
    null => string.Empty,
    IFormattable formattable => formattable.ToString(null, FormatProvider),
    _ => value.ToString()
};
```